src/hooks/mxapp.js

```diff
@@ -0,0 +1,28 @@
+const Hook = require("../hook");
+
+const MXCtrlResponseHandler = require("../handlers/mxapp");
+const MXAppEvent = require("../events/mxapp");
+
+const parseMXAppTrigger = trigger =>
+    (({
+        source,     //an InstanceName
+        message     //a string
+    }) => {
+        const events = [];
+
+        if (message == "UP")
```

Comment:
i'd recommend you not check for this here, but rather in the handler's `matches` predicate. this would make it easier to handle other appEvents in the future

Reply:
That's what that's for?